app/src/components/Map/Timebar.jsx

```diff
@@ -467,6 +471,13 @@ class Timebar extends Component {
   }
 
   render() {
+    const dateFormat = 'DD MMM YYYY';
+    const startDateText = isMobile.phone || isMobile.tablet ? ' start' : 'start date';
```

Comment:
With media queries you have to use widths in px. There's no guarantee for us to know what device is the user using this way. With this library, it reads the User Agent so we are sure about the device the user is using.